torchgeo/datasets/worldstrat.py

```diff
@@ -0,0 +1,368 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""WorldStrat Dataset."""
+
+import os
+from collections.abc import Callable, Sequence
+from glob import glob
+from typing import ClassVar
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import rasterio
+import torch
+from matplotlib.figure import Figure
+from PIL import Image
+from torch import Tensor
+
+from .errors import DatasetNotFoundError
+from .geo import NonGeoDataset
+from .utils import (
+    Path,
+    array_to_tensor,
+    check_integrity,
+    download_and_extract_archive,
+    download_url,
+    extract_archive,
+    percentile_normalization,
+)
+
+
+class WorldStrat(NonGeoDataset):
+    """WorldStrat dataset.
+
+    `WorldStrat <https://worldstrat.github.io/>`_ is a multi-modal dataset covering nearly 10,000km2 of matched high and low resolution
```

Comment:
```suggestion
    `WorldStrat <https://worldstrat.github.io/>`__ is a multi-modal dataset covering nearly 10,000km2 of matched high and low resolution
```
Otherwise the table in the docs will link to this website instead of to our class